app/src/main/kotlin/com/wire/android/ui/home/conversationslist/common/ConversationItemFactory.kt

```diff
@@ -284,6 +286,33 @@ fun PreviewGroupConversationItemWithNoBadges() {
     )
 }
 
+@Preview
+@Composable
+fun PreviewGroupConversationItemWithLastDeletedMessage() {
+    ConversationItemFactory(
+        conversation = ConversationItem.GroupConversation(
+            "groupName looooooooooooooooooooooooooooooooooooong",
+            conversationId = QualifiedID("value", "domain"),
+            mutedStatus = MutedConversationStatus.AllAllowed,
+            lastMessageContent = UILastMessageContent.SenderWithMessage(
+                "John".toUIText(),
+                UIText.StringResource(R.string.deleted_message_text),
+                ": ${MarkdownConstants.NON_BREAKING_SPACE}"
```

Comment:
on tests we have an extra space between `:` and `${Markdown...` is this expected? As in the normal implementation there is no space